<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-06-15T11:18:42 IST" name="register_Suite" finished-at="2025-06-15T11:19:27 IST" duration-ms="44191">
    <groups>
      <group name="dataDriven">
        <method signature="login_withddt.loginddt(java.lang.String,java.lang.String)[pri:0, instance:testcases.login_withddt@7530ad9c]" name="loginddt" class="testcases.login_withddt"/>
      </group> <!-- dataDriven -->
    </groups>
    <test started-at="2025-06-15T11:18:42 IST" name="Test" finished-at="2025-06-15T11:19:27 IST" duration-ms="44191">
      <class name="testcases.login_withddt">
        <test-method is-config="true" signature="setup(java.lang.String,java.lang.String)[pri:0, instance:testcases.login_withddt@7530ad9c]" started-at="2025-06-15T11:18:43 IST" name="setup" finished-at="2025-06-15T11:18:55 IST" duration-ms="12575" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[br]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="loginddt(java.lang.String,java.lang.String)[pri:0, instance:testcases.login_withddt@7530ad9c]" started-at="2025-06-15T11:18:58 IST" name="loginddt" data-provider="LoginData" finished-at="2025-06-15T11:19:04 IST" duration-ms="6196" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[7032753299]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[pass]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginddt -->
        <test-method signature="loginddt(java.lang.String,java.lang.String)[pri:0, instance:testcases.login_withddt@7530ad9c]" started-at="2025-06-15T11:19:04 IST" name="loginddt" data-provider="LoginData" finished-at="2025-06-15T11:19:12 IST" duration-ms="8393" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[7897798797989]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[fail]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginddt -->
        <test-method signature="loginddt(java.lang.String,java.lang.String)[pri:0, instance:testcases.login_withddt@7530ad9c]" started-at="2025-06-15T11:19:12 IST" name="loginddt" data-provider="LoginData" finished-at="2025-06-15T11:19:20 IST" duration-ms="8299" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[123]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[fail]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginddt -->
        <test-method signature="loginddt(java.lang.String,java.lang.String)[pri:0, instance:testcases.login_withddt@7530ad9c]" started-at="2025-06-15T11:19:20 IST" name="loginddt" data-provider="LoginData" finished-at="2025-06-15T11:19:21 IST" duration-ms="777" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[1234567890]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[fail]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[OTP logo found for invalid number: 1234567890]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: OTP logo found for invalid number: 1234567890
at org.testng.Assert.fail(Assert.java:111)
at testcases.login_withddt.loginddt(login_withddt.java:40)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginddt -->
        <test-method is-config="true" signature="tearup()[pri:0, instance:testcases.login_withddt@7530ad9c]" started-at="2025-06-15T11:19:26 IST" name="tearup" finished-at="2025-06-15T11:19:27 IST" duration-ms="216" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearup -->
      </class> <!-- testcases.login_withddt -->
    </test> <!-- Test -->
  </suite> <!-- register_Suite -->
</testng-results>
